neuromation/cli/root.py

```diff
@@ -79,11 +85,28 @@ def client(self) -> Client:
         return self._client
 
     async def init_client(self) -> None:
-        self._factory = Factory(path=self.config_path)
+        trace_configs: Optional[List[aiohttp.TraceConfig]]
+        if self.trace:
+            trace_configs = [self._create_trace_config()]
+        else:
+            trace_configs = None
```

Comment:
Would not be simpler to set `trace_configs = []` here and drop `Optional` from annotations?

Reply:
The only reason is that `ClientSession` has default `None` for `trace_configs`. The empty list works too, though.